int32[] football_xyxy
int32[] goal_xyxy
int32[] net_xyxy
int32[] robot_xyxy
int32[] penalty_mark_xyxy
int32[] center_circle_xyxy